docs/legacy/sourcemaps.asciidoc

```diff
@@ -104,19 +103,18 @@ module.exports = {
 <1> If you're using a different method of defining `serviceVersion`, you can set it here.
 
 [float]
-[[sourcemap-rum-upload]]
-=== Upload the source map to APM Server
+[[sourcemap-rum-configure]]
+=== Configure the {kib} endpoint in APM Server
 
-The APM Server exposes a <<sourcemap-api,source map endpoint>> for uploading source maps.
-Source maps are cached in memory for as long as the <<rum-sourcemap-cache,cache expiration>> setting indicates.
-Source maps must be attached as a `file upload` and you can configure an API key or secret token to restrict uploads.
+include::./tab-widgets/kibana-endpoint-widget.asciidoc[]
 
-Before uploading a source map, ensure that:
+[float]
+[[sourcemap-rum-upload]]
+=== Upload the source map to Kibana
 
-1. RUM support is enabled in APM Server by setting <<rum-enable,`apm-server.rum.enabled`>> to `true`.
-2. <<config-sourcemapping-enabled,`apm-server.rum.source_mapping.enabled`>> is `true` (the default).
-This means that if a source map has previously been uploaded,
-source mapping will automatically be applied to all incoming transactions and errors.
+Kibana exposes a {kibana-ref}/rum-sourcemap-api.html[source map endpoint] for uploading source maps.
+Source maps can be uploaded as a string, or as a file upload.
+Before uploading a source map, ensure that RUM support is enabled in the APM integration
```

Comment:
That's actually not important for the upload; only for accepting RUM events on the APM Server intake.

Reply:
Ah! Thanks, I'll make that clear.